.github/workflows/version.yml

```diff
@@ -0,0 +1,71 @@
+name: Nightly dbt-version check and update.
+on:
+  push:
```

Comment:
```suggestion
```
I think we can just run this nightly and when manually triggered, no need to run on every push. This was left over from testing. I just figured out we can test this action locally with `act -W .github/workflows/version.yml --detect-event`. Can you add a comment to the top of the file mentioning that you can test this workflow locally with `act` using that command?

Reply:
removed and added comments